airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -234,7 +244,7 @@ def _export_cmd(self, table, export_dir, input_null_string,
                     input_null_non_string, staging_table, clear_staging_table,
                     enclosed_by, escaped_by, input_fields_terminated_by,
                     input_lines_terminated_by, input_optionally_enclosed_by,
-                    batch, relaxed_isolation):
+                    batch, relaxed_isolation, extra_export_options):
```

Comment:
add the option to pass in extra export arguments that sqoop offers. There are many options that sqoop offers that aren't part of the SqoopOperator and there should be a way to pass those in.

Reply:
👍